tests/FeatureManagement.UnitTests/Microsoft.Omex.Extensions.FeatureManagement.UnitTests.csproj

```
@@ -4,7 +4,8 @@
     <TargetFrameworks>$(UnitTestTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
 
-  <ItemGroup>
+  <!-- TODO 10880196 - System.Linq.Async is needed for .NET 9 but conflicts with built-in support in .NET 10 -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net10.0'">
```

This will break during future updates.
Suggest inverting the comparison.
```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'">
```